package.json

```diff
@@ -22,9 +22,9 @@
   "scripts": {
     "build": "tsc -p tsconfig.build.json",
     "build:watch": "npm run build -- --watch",
-    "build-types": "ts-node src/scripts/buildTypes.ts",
-    "metadata": "ts-node src/scripts/buildMetadata.ts",
-    "adapters-cli": "ts-node src/scripts/index.ts",
+    "adapters-cli": "node lib/main/scripts/index.js",
+    "build-metadata": "npm run adapters-cli build-metadata --",
+    "build-types": "npm run adapters-cli build-types --",
```

Comment:
Moved last two commands to `Commander`.

Changed to use the compiled version, because otherwise it is impossible to run ts-node if there's a TS error anywhere in the code.